Core/Utilities/TimeLeft/MJFTimeLeft.py

```diff
@@ -0,0 +1,93 @@
+########################################################################
+# $Id$
+########################################################################
+
+""" The Machine/Job Features TimeLeft utility interrogates the MJF values
+    for the current CPU and Wallclock consumed, as well as their limits.
+"""
+
+from DIRAC import gLogger, S_OK, S_ERROR
+
+__RCSID__ = "$Id$"
+
+import os, re, time, urllib
+
+class MJFTimeLeft:
```

Comment:
This (and the other stylistic comments that have been made) is the result of making as few changes as possible compared to the style in the existing TimeLeft code for the various batch systems.